core/src/io/anuke/mindustry/world/blocks/distribution/ArmoredConduit.java

```diff
@@ -0,0 +1,46 @@
+package io.anuke.mindustry.world.blocks.distribution;
+
+import io.anuke.arc.Core;
+import io.anuke.arc.graphics.g2d.Draw;
+import io.anuke.arc.graphics.g2d.TextureRegion;
+import io.anuke.mindustry.type.Liquid;
+import io.anuke.mindustry.world.Block;
+import io.anuke.mindustry.world.Edges;
+import io.anuke.mindustry.world.Tile;
+
+public class ArmoredConduit extends Conduit{
```

Comment:
This class needs to be re-formatted w/o spaces. Also has extra indent before `capRegion`.